lib/helper.c

```diff
@@ -935,7 +935,7 @@ ipmi_start_daemon(struct ipmi_intf *intf)
 		exit(1);
 	}
 	ret = dup(fd);
-	if (ret != STDOUT_FILENO) {
+	if (ret != STDERR_FILENO) {
```

Comment:
Looks like nobody has ever used this code because you're the first to notice this.
Thanks.

Reply:
Please change the commit message to
```
helper: Fix stderr file descriptor

The third file descriptor is STDERR_FILENO, not STDOUT_FILENO
```